adapter/slack.py

```diff
@@ -29,18 +31,26 @@ async def ask(self, request: Request):
         else:
             raise HTTPException(status_code=400, detail="Missing parameter in the request.")
         
-        question = f"{question} \n\n<@{user_id}>"
-        
+        question = f"<@{user_id}> asked: \n\n\"{question}\" "

```

Comment:
Is there a specific reason for changing the format?

Reply:
This is purely subjective but I just thought it looked nicer.